tests/transforms/test_mitigate.py

```diff
@@ -513,6 +523,15 @@ def test_polyfit(self):
         coeffs = qml.transforms.mitigate._polyfit(x, y, 2)
         assert qml.math.allclose(qml.math.squeeze(coeffs), [3, 2, 1])
 
+    def test_exponentialfit(self):
```

Comment:
Could parametrize the test (different values for the numeric parameters)?

Reply:
Good idea. I added tests for both increasing and decreasing exponentials in https://github.com/PennyLaneAI/pennylane/pull/5972/commits/32af13c85c3c160844ba23013996c89b9c58baf8.